src/script/arch-update.sh

```diff
@@ -10,6 +10,11 @@ _name="Arch-Update"
 version="1.15.0"
 option="${1}"
 
+if [ -z "$ARCH_UPDATE_STATE"]
```

Comment:
Are we able to use the already existing [statedir mechanism](https://github.com/Antiz96/arch-update/blob/main/src/script/arch-update.sh#L85) implemented in users' $XDG_STATE_HOME dir instead of using `/var/lib/arch-update/state`?

That will only dropping the change in the Makefile and the addition of the "res/state" file as well as avoiding having another file write-able by world (666 mode).

Reply:
Addressed in https://github.com/Antiz96/arch-update/pull/148/commits/98ec9afad34990ef522201125d96d4d8caab9543